src/Chat.tsx

```diff
@@ -160,21 +165,10 @@ export class Chat extends React.Component<ChatProps, State> {
         });
     }
 
-    componentDidMount() {
-        setTimeout(() => {
-            this.setState({isPersistentMenuOpen: true});
-        }, 1000);
-        
-        // Now that we're mounted, we know our dimensions. Put them in the store (this will force a re-render)
-        this.setSize();
-
+    startConnection = () => {
```

Comment:
you should check for !this.botConnection in this function, since is called from multiple places